apps/cloud/src/api/user.ts

```diff
@@ -0,0 +1,47 @@
+/**
+ * Created by acunningham on 22/05/17.
+ */
+import Store from '@raincatcher/store';
+import webapi, {ApiService, StoreApiService, WebApiConfig}  from '@raincatcher/webapi';
+import * as express from 'express';
+
+
+// Define User datatype
+interface User {
+  id: string,
+  username: string,
```

Comment:
We might merge username and id, since we only need one unique identifier to correlate with keycloak.

I understand this is here for now because of the constraint in the sample Store.